### PR TITLE
Improve Web interface

### PR DESCRIPTION
**Pull request by KagurazakaNyaa**

### CODE REVIEW

**Comment by azlux** [comment]
Thank a lot for the improvement. I will test that ASAP.

File: configuration.default.ini [hunk 1]
```diff
@@ -11,8 +11,8 @@ username = botamusique
 comment = Hi, I'm here to play radio, local music or youtube/soundcloud music. Have fun !
 volume = 0.1
 admin = User1;User2; # Allow user to kill the bot
-music_folder = /home/azlux/botamusique/music_folder/
-tmp_folder = /tmp/
+music_folder = music_folder/
+tmp_folder = tmp/
```
Comment (Lartza) [inline configuration.default.ini]: Pretty sure this was /tmp/ deliberately

File: templates/index.html [hunk 2]
```diff
@@ -1,126 +1,237 @@
...
 <!DOCTYPE html>
+
 <head>
     <meta charset="UTF-8">
-    <title></title>
-    <link rel="stylesheet" href="{{ url_for('static', filename='index.css') }}">
+    <title>botamusique web interface</title>
+    <link rel="stylesheet" href="/static/css/bootstrap.min.css">
     <META HTTP-EQUIV="Pragma" CONTENT="no-cache">
     <META HTTP-EQUIV="Expires" CONTENT="-1">
```
Comment (Lartza) [inline templates/index.html]: These are invalid and should be removed.

File: templates/index.html [hunk 3]
```diff
@@ -1,126 +1,237 @@
...
+                <div class="card">
+                    <div class="card-header">
+                        <h5 class="card-title">Upload File</h5>
+                    </div>
+                    <div class="card-body">
+                        <form action="./upload" method="post" enctype="multipart/form-data">
+                            <div class="input-group">
+                                <div class="custom-file">
+                                    <input type="file" name="file" class="custom-file-input" id="uploadSelectFile"
+                                        aria-describedby="uploadSubmit" value="Browse Music file" />
```
Comment (Lartza) [inline templates/index.html]: "value" is not accepted here

File: templates/index.html [hunk 4]
```diff
@@ -1,126 +1,237 @@
...
+                    </div>
+                    <div class="card-body">
+                        <form method="post">
+                            <label>Add Radio URL :</label>
+                            <input class="form-control" type="text" name="add_radio">
+                            <button type="submit" class="btn btn-primary">Add Radio</button>
+                        </form>
+                    </div>
+                </div>
+            </div>
```
Comment (Lartza) [inline templates/index.html]: Make these responsive so they collapse on a mobile screen?

**Pull request merged**